### PR TITLE
Improve the issue templates

### PR DESCRIPTION
**Pull request by RaptaG**
Should be more useable and generally better

### CODE REVIEW

**Comment by RaptaG** [comment]
@KRTirtho can you merge?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,94 +1,71 @@
...
-        - "Official Website (spotube.netlify.app)"
-        - "Github Release (Binary)"
-        - "Github Actions (Nightly Binary)"
+        - "Website (spotube.netlify.app)"
+        - "GitHub Releases (Binary)"
+        - "GitHub Actions (Nightly Binary)"
         - "F-Droid (Android)"
         - "Arch User Repository (AUR)"
-        - "Flathub (Flatpak)"
+        - "FlatHub (Flatpak)"
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:60]: No, it's actually Flathub

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -1,94 +1,71 @@
...
         - "WinGet (Windows)"
         - "Scoop (Windows)"
     validations:
       required: true
-
   - type: textarea
     attributes:
       label: Additional information
-      description: Anything else you'd like to add?
+      description: Anything else you'd like to add or include?
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:69]: `Add` and `include` same thing... No need to repeat twice

File: .github/ISSUE_TEMPLATE/new_feature.yml [hunk 3]
```diff
@@ -1,53 +1,37 @@
 name: ✨ New Feature
-assignees:
-  - KRTirtho
 description: Suggest an idea
-title: "[Feature] <title>"
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/new_feature.yml:5]: Why remove this? Ok, just replace the `<title>` with `<your-issue-title>` part

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -1,94 +1,71 @@
...
+        1. I opened the app
+        2. I did this
+        3. And that
+        4. Then this happened
     validations:
-      required: false
+      required: true
   - type: input
-    attributes:
-      label: Spotube Version
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: Why remove this? Knowing the version is a primary requirement

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -1,94 +1,71 @@
...
-      description: In which version of Spotube did you encounter this bug?
-      placeholder: e.g. v2.3.0
-  - type: dropdown
-    attributes:
+      attributes:
       label: Operating System
-      description: In which OS you're experiencing the bug?
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: Same here. Why remove this? We don't want BSD users trying Spotube on BSD but failing and creating an issue here not knowing we don't support BSD :joy:

Comment (KRTirtho) [reply .github/ISSUE_TEMPLATE/bug_report.yml:48]: Keep the options

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -1,94 +1,71 @@
...
-        - "Linux"
-        - "Windows"
-        - "Mac OS"
+      description: The OS in which you used Spotube to face the issue.
+      placeholder: Android, Linux, macOS or Windows? Make sure to include the version too.
     validations:
       required: true
   - type: input
     attributes:
-      label: Linux Distribution (if applicable)
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:59]: This is really necessary. Don't remove it. Different Linux distro behaves differently. So knowing which distro is causing the issue is an absolute necessity

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -1,94 +1,71 @@
...
-      description: Tell us in what Linux Distribution you're experiencing the bug.
-      placeholder: Arch Linux
-    validations:
-      required: false
-  - type: input
-    attributes:
-      label: Operating System Version
```
Comment (KRTirtho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]: OS Version is needed too

**Review by KRTirtho — CHANGES_REQUESTED** [review]

**Comment by KRTirtho** [comment]
@RaptaG please apply this changes and please add the "Logs file" part

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:60]: 💀

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:69]: Eh, alright

File: .github/ISSUE_TEMPLATE/new_feature.yml (hunk 3 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/new_feature.yml:5]: Because some people didn't understand it and it doesn't look nice

**Comment by RaptaG** [comment]
I merged some sections together

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:59]: We can understand this from the installation source for most of the times

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:66]: I merged it in the OS section

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:42]: I didn't remove this, did I?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -1,94 +1,71 @@
...
-      description: Please search to see if an issue already exists for the bug you encountered.
+      description: Make sure to check if this issue is a duplicate.
       options:
         - label: I have searched the existing issues
           required: true
   - type: textarea
     attributes:
       label: Current Behavior
-      description: Tell us what you're experiencing concisely.
+      description: Type what you are experiencing currently.
```
[the remark this answers is not in the document]
Comment (RaptaG) [reply .github/ISSUE_TEMPLATE/bug_report.yml:17]: 👍🏽

File: .github/ISSUE_TEMPLATE/new_feature.yml (hunk 3 above)
Comment (KRTirtho) [reply .github/ISSUE_TEMPLATE/new_feature.yml:5]: That's just some people. Not everyone...
That's why, `your-issue-title` or `write-title-here` etc will help

**Pull request merged**